packages/babel-parser/src/parser/expression.ts

```diff
@@ -1730,13 +1730,12 @@ export default abstract class ExpressionParser extends LValParser {
     pattern: string;
     flags: N.RegExpLiteral["flags"];
   }) {
-    const node = this.parseLiteral<N.RegExpLiteral>(
-      value.value,
-      "RegExpLiteral",
-    );
+    const node = this.startNode<N.RegExpLiteral>();
+    this.addExtra(node, "raw", this.input.slice(node.start, this.state.end));
```

Comment:
This may also be removed (probably Babel 8)

Reply:
I would keep it just for consistency with other literals